test/e2e/antreaipam_test.go

```diff
@@ -32,80 +32,84 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	utilnet "k8s.io/utils/net"
 
-	crdv1alpha2 "antrea.io/antrea/pkg/apis/crd/v1alpha2"
+	crdv1beta1 "antrea.io/antrea/pkg/apis/crd/v1beta1"
```

Comment:
maybe we should have 1 e2e test which validates CR conversion (CR is created with old API version, then we retrieve the CR with the new version, and vice versa).

Reply:
Will check this part.

Reply:
Done.